modules/cache/cache.ts

```diff
@@ -106,25 +108,28 @@ export const getCacheInfo = async (name: string, network: SupportedNetworks): Pr
 export const cacheGet = async (
   name: string,
   network?: SupportedNetworks,
-  expiryMs?: number
+  expiryMs?: number,
+  extendedTTL = false,
+  method: 'GET' | 'HGET' = 'GET',
```

Comment:
We currently only store individual data into individual cache keys, `HSET` provides a way of creating key-value pairs of data into a single cache key, allowing us to store all of the individual Poll objects into a single cache key. `HGET` is the way you fetch individual items from that cache key by using the specific item key without having to fetch the whole dataset stored in the cache key.

When migrating the API to use, process, and store in the cache the polls from the new aggregated polls file (<https://raw.githubusercontent.com/hernandoagf/community/master/governance/polls/meta/polls.json>), I found that using the `SET` method to store the ~900 polls in the cache at the same time would cause the API call to lag for several seconds, while `HSET` would store the data much faster